tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradePlanner.java

```diff
@@ -58,4 +61,10 @@ public UpgradePlan newUpgradePlan(
 
 	public void startUpgradePlan(UpgradePlan upgradePlan);
 
+	public void updateUpgradePlanOutlines(List<String> outlines);
+
+	public final List<String> defaultUpgradePlanOutlines = Arrays.asList(
+		"liferay-code-upgrade-plan," +
+			"http://localhost:8080/web/guest/docs/7-2/reference#upgrading-code-to-liferay-portal-ce-72");
```

Comment:
we have to set an internal url until we get a public office link. And I add the preference page to allow to set custom outline link.